ludwig/schema/features/utils.py

```diff
@@ -0,0 +1,119 @@
+from ludwig.decoders.registry import get_decoder_classes
+from ludwig.encoders.registry import get_encoder_classes
+from ludwig.schema import utils as schema_utils
+from ludwig.utils.registry import Registry
+
+input_type_registry = Registry()
+output_type_registry = Registry()
+
+
+def register_input_feature(name: str):
+    def wrap(cls):
+        input_type_registry[name] = cls
+        return cls
+
+    return wrap
+
+
+def register_output_feature(name: str):
+    def wrap(cls):
+        output_type_registry[name] = cls
+        return cls
+
+    return wrap
+
+
+def update_encoders(feature_props, feature_type):
```

Comment:
A bit confused why these two update functions are necessary. Aren't the `encoder` props on the feature schemas declared with calls to `get_encoder_classes`/`get_decoder_classes` already?

Reply:
gotcha! please add docstring or comment where it's called with this note

Reply:
Will do!